client/home.js

```diff
@@ -205,15 +205,61 @@ Template.home.helpers({
   }
 });
 
+var getcheckBoxes = function(){
+  var boxes = [
+    "deepstream",
+    "youtube",
+    "bambuser",
+    "ustream",
+    "twitch"
+  ];
+
+  var checked  = boxes.filter(function(elem){
+    var id = elem + "-checkbox";
+    document.getElementById(id).onchange = getcheckBoxes;
+    return document.getElementById(id).checked;
+  });
+
+  if(checked.length == 0){
+    console.log("No boxes were selected");
+    return;
+  }
+  boxesDep.changed();
+  Session.set("checkedBoxes", checked);
+};
+
 var getHomepageStreamSearchResults = function() {
+  boxesDep.depend();
```

Comment:
`boxesDep` shouldn't be necessary, since the Session variable "checkedBoxes" is already reactive (so this function will already rerun when it changes)